### ISSUE TITLE
🐛 Bug: Accessibility issue: missing lang on <html>

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [SquiggleTools/boston-ts-website#306]
### Bug Report Checklist

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have pulled the latest `main` branch of the repository.
- [X] I have [searched for related issues](https://github.com/JoshuaKGoldberg/boston-ts-website/issues?q=is%3Aissue) and found none that matched my issue.

### Expected

There should be no accessibility issues -programmatically detectable or otherwise- on the website.

### Actual

Running aXe via Lighthouse reveals that the `<html>` is missing a `lang`.

> `<html>` element does not have a `[lang]` attribute
>
> If a page doesn't specify a lang attribute, a screen reader assumes that the page is in the default language that the user chose when setting up the screen reader. If the page isn't actually in the default language, then the screen reader might not announce the page's text correctly. [Learn more about the lang attribute](https://dequeuniversity.com/rules/axe/4.9/html-has-lang).

### Additional Info

💖

**Comment by astorije** [comment]
Nice catch! https://github.com/SquiggleTools/boston-ts-website/pull/308

**Comment by astorije** [comment]
I know Lighthouse, but what is aXe?!

**Comment by JoshuaKGoldberg** [comment]
aXe is the core library that checks for accessibility issues, it's awesome! https://github.com/dequelabs/axe-core
